content/blog/2021/machine-learning.md

```diff
@@ -0,0 +1,63 @@
+
+<p class="text-center">
+  <img src="/content/images/ml-article-2021-Relations.png" alt="Relations between AI, ML, DL, Big Data and Data Science"/>
+</p>
+
+Yes, and machine learning is one of the most fascinating aspects of Artificial Intelligence.
+
+## So what is machine learning?
+
+Machine learning leverages the power of statistical modelling to learn patterns in data that can be leveraged to predict outcomes from previously unseen data.
+
+Imagine a situation, where a business wants to increase the profits for the next quarter by selling a diverse range of products. The question lies, which product could be the most profitable, at which period of time or what factors would affect the product sales.  .
+
+One could start by observing patterns in the data from last year’s product sales and growth seen by the company in the same quarter. From that shortlist the best products manually. 
+
+Or, we can automate by working out some metrics to get more rigorous proof, rather than solely working on our intuition. 
+
+This is where machine learning comes to the rescue. It automates the steps that you’ll do while working out the statistical metrics on a larger scale, scaling to hundreds, if not thousands of products, and millions of observations. The end result? Predictions that you can rely on (provided the input data is correct, clean and free from noise).
+
+## Where does Python play its part?
+
+Python being one of the easiest languages to get started with for machine learning applications makes it a clear choice to use it when prototyping your ideas. 
+
+With hundreds of libraries at our disposal like matplotlib, scikit-learn, numpy, pandas and scipy make data visualization, large-scale computation, data manipulation and machine learning accessible like never before. Combine that with a large community base and lots of developments in the field. Machine Learning has quickly become a major conference section where we receive a lot of submissions from both eminent researchers, professionals and students. 
+
+## Talks to look forward to about Machine learning in PyCon India 2021
+
+1.   Reducing technical debt for ML platforms
+    
+2.  Visualisation & Underlined Psychology
+    
+3.  Touches of Python in Prosthetic Vision : New Generation Computer vision
+    
+4.  How To Log ML Experiments
+    
+5.  On Device AI - Deep Learning on Mobile Devices
+  
+## Talks in the past that have been delivered by speakers that truly stand out
+
+1.  Taking a peek under the hood: Interpreting black box models (PyCon India 2020)
+    
+2.  Towards a more transparent AI - Decrypting ML models using LIME (PyCon India 2020)
+    
+3.  Resource Utilization as a Metric for Machine Learning (PyCon India 2020)
+    
+4.  Machine Learning for IoT at the Edge (PyCon India 2019)
+    
+5.  IndicNLP - An open data platform to bring Indian languages to the advancements of NLP. (PyCon India 2019)
```

Comment:
If you can add links to all these talks that would be really helpful as well.

Reply:
Yeah I'll do it ASAP.

Reply:
Done.